src/senaite/core/upgrade/v02_02_000.py

```diff
@@ -214,5 +214,12 @@ def migrate_worksheet_layouts(portal):
         if layout:
             # set the new layout
             obj.setResultsLayout(layout)
+    
+    logger.info("Update default worksheet layout for BikaSetup ...")
+
+    bika_setup = api.get_bika_setup()
```

Comment:
just a minor thing:
we're fading out the `bika` namespace, so better use `setup = api.get_setup()`

Reply:
done